lib/dry/configurable/instance_methods.rb

```diff
@@ -11,9 +11,16 @@ module Configurable
     # @api private
     module Initializer
       # @api private
-      def initialize(*)
-        @config = Config.new(self.class._settings.dup)
-        super
+      if RUBY_VERSION >= "2.7"
+        def initialize(*, **)
+          @config = Config.new(self.class._settings.dup)
+          super
+        end
+      else
+        def initialize(*)
+          @config = Config.new(self.class._settings.dup)
+          super
+        end
```

Comment:
I *think* we can levarage `...` in >= 3.0 too, so:

```ruby
if RUBY_VERSION >= "2.7"
  def initialize(...)
     @config = Config.new(self.class._settings.dup)
     super
  end
end
```

Reply:
1) it's 2.7
2) it'll require using strings and class_eval since it's a syntax change. I'd postpone `...` till 2.6 is EOL